specification/sections/01-flag-evaluation.md

```diff
@@ -513,3 +513,9 @@ see: [error codes](../types.md#error-code), [flag value resolution](./02-provide
 The SDK ensures that if the provider's lifecycle methods terminate with an `error code`, that error code is included in any associated error events and returned/thrown errors/exceptions.
 
 see: [error codes](../types.md#error-code)
+
+#### Requirement 1.7.9
+
+> The client's `provider status` accessor **MUST** indicate `NOT_READY` once the `shutdown` function of the associated provider terminates.
```

Comment:
This is basically the old requirement, but relocated to the API, since providers themselves don't have this field.

This point is very similar to points `1.7.3` to `1.7.5` above.